cnchi/features_info.py

```diff
@@ -135,7 +135,7 @@ def _(message):
               "Apache or Nginx) plus a database server (Mariadb)\n"
               "and PHP."),
     'lts': _("The linux-lts package is an alternative Arch kernel package\n"
-             "based upon Linux kernel 3.14 and is available in the core repository.\n"
+             "based upon Linux kernel 4.4 and is available in the core repository.\n"
```

Comment:
Thanks for the PR! :smiley: Let's go ahead and remove this entire line instead. There is no need to mention the version here IMO. Be sure to add a period at the end of line `137.